src/controllers/controller.line.js

```diff
@@ -108,14 +111,17 @@ export default DatasetController.extend({
 			const properties = {
 				x,
 				y,
-				skip: isNaN(x) || isNaN(y)
+				skip: isNaN(x) || isNaN(y),
+				stop: i > 0 && (parsed.x - prevParsed.x) > maxGapLength
```

Comment:
Maybe we could add `skip` and `stop` to `properties` only if true. I'm wondering if it would improve performance to make the object smaller

Reply:
Its actually bad for performance: https://codereview.stackexchange.com/questions/28344/should-i-put-default-values-of-attributes-on-the-prototype-to-save-space/28360#28360